io.github.pemsley.coot.yaml

```diff
@@ -0,0 +1,511 @@
+app-id: io.github.pemsley.coot
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+command: coot
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+  - --filesystem=host
+  - --share=network
+
+modules:
+  - name: fftw2
+    sources:
+      - type: archive
+        url: https://fftw.org/fftw-2.1.5.tar.gz
+        sha256: f8057fae1c7df8b99116783ef3e94a6a44518d49c72e2e630c24b689c6022630
+    buildsystem: autotools
+    config-opts:
+      - --enable-shared
+      - --enable-float
+      - --disable-static
+    build-commands:
+      - |
+        if [ "$(uname -m)" = "aarch64" ]; then
+          sed -i 's/-flat_namespace -undefined suppress/-undefined dynamic_lookup/g' configure
+        fi
+        ./configure --prefix=/app --enable-shared --enable-float --disable-static
+        make
+        make install
+
+  - name: mmdb2
+    sources:
+      - type: archive
+        url: https://www2.mrc-lmb.cam.ac.uk/personal/pemsley/coot/dependencies/mmdb2-2.0.22.tar.gz
+        sha256: a9646933ce9f34633e1ae4901f2383af0e5318d6490851328f5b6aa06195ab51
+    buildsystem: autotools
+    config-opts:
+      - --enable-shared
+      - --disable-static
+    build-options:
+      env:
+        CXXFLAGS: "-g -O2"
+    build-commands:
+      - |
+        ./configure --prefix=/app --enable-shared --disable-static
+        make
+        make install
+
+  - name: libccp4
+    sources:
+      - type: archive
+        url: https://github.com/cctbx/ccp4io/archive/b58c4fb68902e4e6a58f4a585d0722e542516076.tar.gz
+        sha256: f1edc5a830cd4a078eae700e14b1d89612fb7a12318094363642340aafe41af6
+    buildsystem: simple
+    build-options:
+      env:
+        CPPFLAGS: "-I${FLATPAK_BUILDIR}/libccp4/lib"
+        LDFLAGS: "-L${FLATPAK_DEST}/lib"
+    build-commands:
+      - |
+        cd libccp4
+        ./configure --prefix="/app" --enable-shared --disable-static --disable-fortran
```

Comment:
buildsystem autotools

Reply:
I need change directory to `libccp4` at first.

Reply:
Which buildsystem should be chosen in the case additional commands are needed(In this case `cd libccp4`)?